sdk/servicebus/azure-servicebus/README.md

```diff
@@ -19,6 +19,7 @@ of this package please look at our [migration guide to move from Service Bus V0.
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```

Comment:
We actually want to keep the message that we have about the end of Python 2.7 support.

If you take a look at the issue we refer to here (https://github.com/Azure/azure-sdk-for-python/issues/20691), you can see that we're announcing the end of support for Python 2.7. We don't usually have big announcements like this for dropping Python 3.x versions, b/c Python 2.7 is a specific/special case. You can read more about this [[here](https://www.python.org/doc/sunset-python-2/)]